src/library/scala/collection/immutable/BitSet.scala

```diff
@@ -37,6 +37,8 @@ sealed abstract class BitSet
 
   override def unsorted: Set[Int] = this
 
+  def foo = 42
```

Comment:
This causes a MiMa error so you can see the result of these changes in the log (if everything goes according to plan).